project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.2.6
+sbt.version=1.4.7
```

Comment:
any reason for 1.4.7 instead of 1.5.0?

Reply:
I was just referencing what we were using in the rifd project.